Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -459,7 +459,7 @@ private void RunMatchExpression(FlexMatch binary64DecodedMatch, AnalyzeContext c
 
                 Regex regex =
                     CachedDotNetRegex.GetOrCreateRegex(matchExpression.ContentsRegex,
-                                                      RegexDefaults.DefaultOptionsCaseInsensitive);
+                                                      RegexDefaults.DefaultOptionsCaseSensitive);
```

Comment:
>DefaultOptionsCaseSensitive [](start = 68, length = 27)

we were not considering casing here, which is wrong. Since we might have a regex that is case sensitive.

Reply:
Ouch! by the way, the spacing is wrong on this argument, it should be moved one column to the right. 

---
In reply to: [600039240](https://github.com/microsoft/sarif-pattern-matcher/pull/326#discussion_r600039240) [](ancestors = 600039240)